rebar.config

```diff
@@ -24,7 +24,7 @@
 
 {deps, [
     {cowlib,      ".*", {git, "https://github.com/extend/cowlib.git",           {tag, "1.0.0"}}},
-    {leo_commons, ".*", {git, "https://github.com/leo-project/leo_commons.git", {tag, "1.1.2"}}},
+    {leo_commons, ".*", {git, "https://github.com/windkit/leo_commons.git", {branch, "develop"}}},
```

Comment:
Regarding ``leo_commons``, I've merged your request. You could modify the row of leo_commons in the deps as below:
```erlang
{leo_commons, ".*", {git, "https://github.com/leo-project/leo_commons.git", {branch, "develop"}}},
```

* https://github.com/leo-project/leo_commons/pull/4